collections/JAXA_wq_chla.yaml

```diff
@@ -0,0 +1,28 @@
+Name: JAXA_wq_chla
+Description: "water quality Chlorophil a anomalies"
+AdditionalInfoExternal:
+    Title: Additional info
+    Path: jaxa_wq_chla/README.MD
+Resolution:
+GeographicalCoverage: 12.0, 33.85, 140.25, 45.8
+TemporalAvailability: 2018-01-06 - 2021-04-17
+UpdateFrequency: weekly
+BandInformation:chla
+Contact: "[JAXA]"
+Provider:"[JAXA]"
+ManagedBy:"[ESA](http://www.esa.int/)"
```

Comment:
Requires space after the colon to separate key value pairs.

Reply:
Done